packages/core/test/insured-bridge/e2e/helpers/ArtifactsHelper.js

```diff
@@ -18,7 +18,7 @@ function getOptimismArtifact(contractName, ovm = false) {
 
 function getLocalArtifact(contractName, ovm = false) {
   const coreContractsPath = findPathToRootOfPackage("@uma/core");
-  const artifactsPath = ovm ? `${coreContractsPath}/artifacts-ovm` : `${coreContractsPath}/artifacts`;
+  const artifactsPath = ovm ? `${coreContractsPath}artifacts-ovm` : `${coreContractsPath}artifacts`;
```

Comment:
why isn't the slash needed? I would have guessed otherwise

Reply:
for some reason between when I set this up and now the call to `findPathToRootOfPackage` now contains a slash. The only reasoning to why this happened is that a package was updated under the hood and impacted the behavior 🤷 . Honestly not really sure how it worked before if that assumption is not true.

Reply:
Actually, I dont think we were importing `ovm` artifacts before (only non-ovm artifacts locally...but even then the slash change does not compute